torchmetrics/metric.py

```diff
@@ -450,7 +459,7 @@ def _load_from_state_dict(
             state_dict, prefix, local_metadata, True, missing_keys, unexpected_keys, error_msgs
         )
 
-    def _filter_kwargs(self, **kwargs):
+    def _filter_kwargs(self, **kwargs: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
```suggestion
    def _filter_kwargs(self, **kwargs: Any) -> Dict[str, Any]:
```

kwargs are always of type Dict[str, X] so they are only annotated with X